Gem/Code/Source/Components/Multiplayer/GemComponent.cpp

```diff
@@ -32,15 +32,25 @@ namespace MultiplayerSample
 
             // Tick on every frame.
             m_clientAnimationEvent.Enqueue(AZ::Time::ZeroTimeMs, true);
+
+            Physics::RigidBodyRequestBus::Event(GetEntityId(), &Physics::RigidBodyRequestBus::Events::DisablePhysics);
+            // Physical bodies take time to enable after entity activation, so sign up for physics activation and disable it
+            Physics::RigidBodyNotificationBus::Handler::BusConnect(GetEntityId());
         }
     }
 
     void GemComponent::OnDeactivate([[maybe_unused]] Multiplayer::EntityIsMigrating entityIsMigrating)
     {
+        Physics::RigidBodyNotificationBus::Handler::BusDisconnect();
         m_clientAnimationEvent.RemoveFromQueue();
         m_networkLocationHandler.Disconnect();
     }
 
+    void GemComponent::OnPhysicsEnabled([[maybe_unused]] const AZ::EntityId& entityId)
+    {
+        Physics::RigidBodyRequestBus::Event(GetEntityId(), &Physics::RigidBodyRequestBus::Events::DisablePhysics);
```

Comment:
Can we add comment about why we don't use the entityId passed?

Reply:
Good point, you can use `entityId` passed, no need to call `GetEntityId()`

Reply:
Done, added a comment